terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -87,57 +87,49 @@ locals {
   }
 
   dragen_wgs_qc_wfl_id = {
-    dev  = "wfl.5cc28c147e4e4dfa9e418523188aacec"
-    prod = "wfl.d6f51b67de5b4d309dddf4e411362be7"
+    dev  = "wfl.23f61cb1baab412a8c37dc93bed6c2af"
```

Comment:
`dev` workflow ID should be `wfl.ff6ca1789f4e4eb0982ea3e01407aca8`

Reply:
Fixed.